sig-network/charter.md

```diff
@@ -0,0 +1,64 @@
+# SIG Network Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Network is responsible for the components, interfaces, and APIs which expose networking capabilities to Kubernetes
```

Comment:
should we give some examples here?

"This includes both the networking control and data paths. Key areas include network service abstractions, service discovery (DNS), service load balancing (L4, L7), network security and identity and cluster connectivity. SIG-NETWORK is also responsible for cross-cutting concerns such as scalability, metrics and monitoring associated with networking components.,"

Reply:
I think the more detailed key areas bit is covered by the "In scope" (and to a lesser extent, the "out of scope") subsections below. Probably no need to explicitly call out particulars like DNS / Services, etc both here and below?

Agreed it's worth calling out the broader areas of scalability, metrics, monitoring explicitly in this section.

Reply:
I was looking at how the other sigs structured their charters.

IMHO this section gives the broad categories and examples (networking capabilities is quite general).
The in code binaries, services seems like reference to actual artifacts?

Reply:
Yep, looking at cluster-lifecycle, they've used this section for a very high level description, but also have a section that we're missing right now which I think covers the examples you're looking for. I'll try modeling it after that and see what we think :)